doc/src/configuration.md

```diff
@@ -1,10 +1,10 @@
 # Configuration
 
-Rustup has a settings file in [TOML](https://github.com/toml-lang/toml) format
-at `${RUSTUP_HOME}/settings.toml`. The schema for this file is not part of the
-public interface for rustup - the rustup CLI should be used to query and set
-settings.
+Rustup has a [TOML](https://github.com/toml-lang/toml) settings file at
+`${RUSTUP_HOME}/settings.toml` (which defaults to `~/.rustup` or
```

Comment:
Do we need to specify which OS it is for?

Reply:
I copied from [the Environment Variables section](https://rust-lang.github.io/rustup/environment-variables.html):
![image](https://user-images.githubusercontent.com/38900226/114790524-47602480-9d5b-11eb-8b1d-a6581c2f4532.png)

The OS isn't specified, so I assumed it was ok.